cmd/interval/interval.go

```diff
@@ -15,10 +15,10 @@
 package interval
 
 import (
-	addinterval "github.com/edgexfoundry-holding/edgex-cli/cmd/interval/add"
-	listinterval "github.com/edgexfoundry-holding/edgex-cli/cmd/interval/list"
-	rminterval "github.com/edgexfoundry-holding/edgex-cli/cmd/interval/rm"
-	updateinterval "github.com/edgexfoundry-holding/edgex-cli/cmd/interval/update"
+	"github.com/edgexfoundry-holding/edgex-cli/cmd/interval/add"
```

Comment:
may be just an artifact of the PR order .. but holding is mentioned here.

Reply:
yes, all these imports that contains "holding" are removed by PR https://github.com/edgexfoundry/edgex-cli/pull/279

Reply:
It looks that the conflicts between PR #279 and this one haven't need resolved correctly. So now the CLI cannot compile. I will open new issue, and will remove a few places where import "holding " is still present